db/schema.rb

```diff
@@ -378,6 +378,7 @@
     t.text "contact"
     t.text "learning_objectives"
     t.string "fields", default: [], array: true
+    t.boolean "visible", defaut: true
```

Comment:
Just wanted to point out that "default" is misspelled here.

Reply:
Good catch. I'm slightly surprised Rails didn't complain about this when loading the schema in the CI